src/Raven.Client/Documents/Commands/Batches/CopyTimeSeriesCommandData.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Session;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+
+namespace Raven.Client.Documents.Commands.Batches
+{
+    public class CopyTimeSeriesCommandData : ICommandData
+    {
+        public CopyTimeSeriesCommandData(string sourceDocumentId, string destinationDocumentId, string changeVector)
+        {
+            if (string.IsNullOrWhiteSpace(sourceDocumentId))
+                throw new ArgumentNullException(nameof(sourceDocumentId));
+            if (string.IsNullOrWhiteSpace(destinationDocumentId))
+                throw new ArgumentNullException(nameof(destinationDocumentId));
+           
+
+            Id = sourceDocumentId;
+            Name = destinationDocumentId;
```

Comment:
Check how it is done in CopyAttachmentCommandData
The property is called 'DestinationId'